CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Additional labels for pre-release and build metadata are available as extensions
 3. Bug fix in `_Measurements_get_table` function (Issue [#421](https://github.com/brightwind-dev/brightwind/issues/421)).
 4. Bug fix in `_Measurements__get_properties` function to handle solar sites (Issue [#453](https://github.com/brightwind-dev/brightwind/issues/453)).
 4. Bug fix in `_Measurements_get_names` function and made this function public (Issue [#450](https://github.com/brightwind-dev/brightwind/issues/450)).
+5. More representative error messages returned in `Shear` function when no valid data is available. Additionally Shear for hour of day or direction sector has now been updated to return np.nan when there is no valid data for that sector or hour (Issue [#205](https://github.com/brightwind-dev/brightwind/issues/205)).
```

Comment:
didn't you update only the shear by time of day to have nan when data is not valid for an hour? Didn't the shear by sector have already this behaviour?